.github/workflows/opensearch_changelog_workflow.yml

```
@@ -0,0 +1,24 @@
+name: OpenSearch Changelog Workflow
+
+on:
+  pull_request_target:
+    types: [opened, edited]
+
+permissions:
+  contents: read
+  issues: write
+  pull-requests: write
+
+jobs:
+  update-changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+      - name: Parse changelog entries and submit request for changset creation
+        # uses: BigSamu/OpenSearch_Changelog_Workflow@main
+        uses: BigSamu/OpenSearch_Changelog_Workflow@feat/manual-changesets
+        with:
+          token: ${{secrets.GITHUB_TOKEN}}
+          changelog_pr_bridge_url_domain: ${{secrets.CHANGELOG_PR_BRIDGE_URL_DOMAIN}}
+          changelog_pr_bridge_api_key: ${{secrets.CHANGELOG_PR_BRIDGE_API_KEY}}
```

I think this is the only change needed for this to work. Without this, i was getting a linting error on my local dev env. Maybe it does not need it but this is the case sensitive name for the variables in your action.

```suggestion
          CHANGELOG_PR_BRIDGE_URL_DOMAIN: ${{secrets.CHANGELOG_PR_BRIDGE_URL_DOMAIN}}
          CHANGELOG_PR_BRIDGE_API_KEY: ${{secrets.CHANGELOG_PR_BRIDGE_API_KEY}}
```